portfolio/src/main/java/com/google/sps/data/CommentDatabase.java

```diff
@@ -24,18 +26,19 @@ public CommentDatabase(DatastoreService datastore) {
   }
 
   /* public wrappper method to return all comments from datastore as an arraylist */
-  public ArrayList<Comment> getCommentsAsArrayList(){
-    this.getCommentsFromQuery();
-    // sort so that comments are displayed from oldest to newest
+  public ArrayList<Comment> getCommentsAsArrayList(int maxCommentsToDisplay){
+    this.getCommentsFromQuery(maxCommentsToDisplay);
```

Comment:
i think we can call this getComments. The "asArrayList" is implied by the return type.

Reply:
Done.